posthog/api/test/dashboards/test_dashboard.py

```diff
@@ -245,15 +245,15 @@ def test_adding_insights_is_not_nplus1_for_gets(self):
                 self.dashboard_api.get_dashboard(dashboard_id, query_params={"no_items_field": "true"})
 
             self.dashboard_api.create_insight({"filters": filter_dict, "dashboards": [dashboard_id]})
-            with self.assertNumQueries(21):
+            with self.assertNumQueries(22):
                 self.dashboard_api.get_dashboard(dashboard_id, query_params={"no_items_field": "true"})
 
             self.dashboard_api.create_insight({"filters": filter_dict, "dashboards": [dashboard_id]})
-            with self.assertNumQueries(21):
+            with self.assertNumQueries(23):
```

Comment:
random fly by that this is not ok 😅

Reply:
Yep, this isn't the actual n+1 case here though, but the organization call from the feature flag.